exercises/practice/robot-name/robot-name.test.ts

```diff
@@ -91,6 +91,38 @@ describe('Robot', () => {
     expect(areSequential(name3, name3)).toBe(true)
   })
 
+  xit('uses all letters', () => {
+    let letters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+    for (let i = 0; i < 1000 - 1; i += 1) {
+      const newRobot = new Robot()
+      for (let j = 0; j < 2; j += 1) {
+        const foundLetter = newRobot.name.charAt(j)
+        if (letters.includes(foundLetter)) {
+          letters = letters.replace(foundLetter, '')
+        }
+      }
+      if (letters.length == 0) break
+    }
+
+    expect(letters).toEqual('')
```

Comment:
```suggestion
    let needles = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
    
    for (let i = 0; i < 1000 - 1; i += 1) {
      const newRobot = new Robot()
      const lettersInName = newRobot.name.slice(0, 2)

      needles = needles.replace(lettersInName[0], '')
      needles = needles.replace(lettersInName[1], '')
      
      if (needles.length === 0) {
        break;
      }
    }

    expect(needles).toEqual('')
```

Reply:
Perhaps like this? Same functionality, but formatted and replaced the for-loop with the imperative one in `.slice`